ansible/roles/composer/tasks/main.yml

```diff
@@ -12,8 +18,11 @@
   when: composer_keep_updated
 
 - name: Install Composer global packages
-  composer: command='global require "{{ item.name }}={{ item.version }}"' prefer_dist=yes working_dir=/
-  with_items: '{{ composer_global_packages }}'
+  composer:
+    command: 'global require "{{ item.name }}={{ item.version }}"'
+    prefer_dist: yes
+    working_dir: /
+  with_items: "{{ composer_global_packages }}"
 
-- include: vagrant.yml
-  when: vagrant
+- include_tasks: vagrant.yml
+  when: vagrant and composer_create_swap
```

Comment:
Is this still required? I thought composer had fixed this?

Reply:
It has as far as I know. I have left it in ans defaulted composer_create_swap to false in case it becomes an issue again.